src/Components/Contributors/AffiliationDisplay.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { FC } from 'react';
+import { Button, Card, CardActions, CardContent, List, ListItem, ListItemText, Typography } from '@material-ui/core';
+import DeleteIcon from '@material-ui/icons/Delete';
+import AddIcon from '@material-ui/icons/Add';
+import styled from 'styled-components';
+
+const StyledAffiliationsWrapper = styled(Card)<{ backgroundColor: string }>`
+  margin-bottom: 2rem;
+  &&.MuiPaper-outlined {
+    border: none;
+    background-color: ${(props) => props.backgroundColor};
+  }
+`;
+
+const StyledCardContent = styled(CardContent)`
+  &&.MuiCardContent-root:last-child {
+    padding-bottom: 0;
+  }
+`;
+
+const StyledCardHeader = styled.div`
+  display: flex;
+  flex-flow: row wrap;
+  align-items: baseline;
+  justify-content: space-between;
+`;
+
+const StyledListItemText = styled(ListItemText)`
+  & .MuiListItemText-primary {
+    font-size: 0.9rem;
+  }
+`;
+
+interface AffiliationDisplayProps {
+  affiliation: { units: string[]; institutionName: string; countryCode?: string };
+  dataTestid: string;
+  backgroundcolor: string;
+  deleteUnitHandleClick?: (unitIndex: number) => void;
```

Comment:
jeg har pleid å kalle disse for handle..xxx..click - føler at det er lettere å lese

Reply:
fikset